src/client/App.js

```diff
@@ -32,6 +33,9 @@ function App() {
         <Route exact path="/">
           <LandingPageContainer />
         </Route>
+        <Route exact path="/order/:id">
+          <ConfirmationPageContainer />
```

Comment:
This is the pass to orderPageContainer. for confirmation page you can do same but like  <Route exact path="/confirmation/:id">